recipe/meta.yaml

```diff
@@ -1,16 +1,22 @@
-{% set version = "14.7.0" %}
+{% set version = "15.3.0" %}
 package:
   name: pyzmq
   version: {{ version }}
 
 source:
   fn: pyzmq-{{ version }}.tar.gz
   # We use the pypi URL as it includes the prepared Cython files.
-  url: https://pypi.python.org/packages/source/p/pyzmq/pyzmq-{{ version }}.tar.gz
-  md5: 87e3abb33af5794db5ae85c667bbf324
+  url: https://pypi.io/packages/source/p/pyzmq/pyzmq-{{ version }}.tar.gz
+  sha256: c4b387071ece45c451c9badf51e02e0a5513fe89abf11dffdf95e81e394d5cf4
+  patches:
+    # Workaround bug in conda-build where compileall prevents installation
+    # due to python3-only asyncio files.
+    # This patch removes those files altogether,
+    # since they aren't importable on py2 anyway.
+    - 0001-remove-asyncio-for-Python-2.patch  # [py27]
```

Comment:
I know we don't build for other Pythons, but could we please change `py27` to `py2k`. This could be useful for people building with older Python 2.x offline.